creator-node/src/utils.js

```diff
@@ -216,8 +220,28 @@ async function rehydrateIpfsDirFromFsIfNecessary (req, dirHash) {
   }
 }
 
+async function fetchCIDFromOtherCreatorNodes (req, multihash, expectedStoragePath) {
+  const libs = req.app.get('audiusLibs')
+  const nodes = await libs.ethContracts.ServiceProviderFactoryClient.getServiceProviderList('creator-node')
+  const endpoints = nodes.map((obj) => obj.endpoint)
+  let foundCID = false
+
+  for (let endpoint of endpoints) {
+    try {
+      await saveFileFromURL(req, endpoint, multihash, expectedStoragePath)
+      foundCID = true
+      break
+    } catch (e) {
+      continue
```

Comment:
im assuming the logic here is to iterate through all the endpoints until one of the endpts has the CID, and hydrates the cnode filesystem with the `saveFileFromURL()` call. if an error occurs, continue on and look at the other endpoints to see if the file exists. is this correct?

either way, could we add a comment here explaining the reason for continuing on if an error is thrown :)